src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java

```diff
@@ -24,6 +24,18 @@
 public class FeedbackQuestionsDbTest extends BaseComponentTestCase {
     private static final FeedbackQuestionsDb fqDb = new FeedbackQuestionsDb();
 
+    private class FeedbackQuestionAttributesWithModifiableAttributes extends FeedbackQuestionAttributes {
```

Comment:
Oops.
Anyway looks like Travis is a bit drunk, they're pulling the old `.yml` file with the two separate configs even after syncing. I might have to close this PR and open a new one.